src/handlers/ws.rs

```diff
@@ -7,18 +7,47 @@ use axum::{
     Extension,
 };
 
-use crate::models::extension_web_socket::ExtensionWebSocket;
+use crate::models::extension_web_socket::{ExtensionWebSocketError, ExtensionWebSocketMatch};
 use futures::{sink::SinkExt, stream::StreamExt};
 
 pub async fn ws_handler(
-    Extension(web_socket): Extension<ExtensionWebSocket>,
-    Path(game_name): Path<String>,
+    Extension(web_socket): Extension<ExtensionWebSocketMatch>,
+    Path(_game_name): Path<String>,
     ws: WebSocketUpgrade,
 ) -> impl IntoResponse {
-    ws.on_upgrade(|socket| handle_socket(socket, game_name, web_socket))
+    ws.on_upgrade(|socket| handle_socket_match(socket, web_socket))
 }
 
-pub async fn handle_socket(ws: WebSocket, _game_name: String, web_socket: ExtensionWebSocket) {
+pub async fn ws_handler_error(
+    Extension(web_socket): Extension<ExtensionWebSocketError>,
+    ws: WebSocketUpgrade,
+) -> impl IntoResponse {
+    ws.on_upgrade(|socket| handle_socket_error(socket, web_socket))
+}
+
+pub async fn handle_socket_error(ws: WebSocket, web_socket: ExtensionWebSocketError) {
+    let cloned_server_rx = web_socket.rx.clone();
+    let (mut ws, mut ws_recv) = ws.split();
+
+    let mut send_task = tokio::spawn(async move {
```

Comment:
Tu je problem ze ked niekto dostane chybovu spravu tak to pride uplne vsetkym

Hodilo by sa to tu nejak odfiltrovat aby kazdemu userovi prisli len jeho errory, trebars ze by ten route bol `/ws/error/:user_uuid` a podla toho, alebo nejak tu informaciu vytiahnut z auth dat